go/config/enclave_config.go

```diff
@@ -52,6 +52,8 @@ type EnclaveConfig struct {
 	ProfilerEnabled bool
 	// MinGasPrice is the minimum gas price for mining a transaction
 	MinGasPrice *big.Int
+	//MessageBusL1 & L2 addresses
+	MessageBusAddresses []*gethcommon.Address
```

Comment:
I think it's better to have separate fields for them:
- MsgBusL1Address
- MsgBusL2Address